pythonforandroid/recipes/sdl2_mixer/toggle_modplug_mikmod_smpeg_ogg.patch

```diff
@@ -1,29 +1,31 @@
---- orig/Android.mk	2016-01-03 07:15:57.000000000 +0100
-+++ patch/Android.mk	2016-04-15 21:28:55.169697882 +0200
-@@ -6,22 +6,22 @@
- 
- # Enable this if you want to support loading MOD music via modplug
- # The library path should be a relative path to this directory.
--SUPPORT_MOD_MODPLUG ?= true
-+SUPPORT_MOD_MODPLUG := false
- MODPLUG_LIBRARY_PATH := external/libmodplug-0.8.8.4
- 
- # Enable this if you want to support loading MOD music via mikmod
- # The library path should be a relative path to this directory.
--SUPPORT_MOD_MIKMOD ?= true
-+SUPPORT_MOD_MIKMOD := false
- MIKMOD_LIBRARY_PATH := external/libmikmod-3.1.12
- 
- # Enable this if you want to support loading MP3 music via SMPEG
- # The library path should be a relative path to this directory.
--SUPPORT_MP3_SMPEG ?= true
-+SUPPORT_MP3_SMPEG := false
```

Comment:
any reason why we don't want to support this things anymore? Is it for licensing issues?

Reply:
@AndreMiras , after a double-check, SUPPORT_MOD_MODPLUG, SUPPORT_MOD_MIKMOD, SUPPORT_MP3_SMPEG were already disabled in 2.0.1 .

Speaking about ogg and Timidity, nice catch, i was thinking that We weren't supporting these features.
If needed, i could test the sdl2_mixer implem with support for timitidy and ogg and push a fix for that.

Reply:
well yes basically if the pull request is just about bumping the version, by default I would not toggle enable/disable things. I mean I would just leave it as it is

Reply:
Also just posted this in discord: SMPEG and MIKMOD seem LGPL, it is really important to always also check licensing issues. LGPL really is problematic not because it's infectious but because of the induced requirements (providing the source and a way to swap it out) which many p4a users will forget about it. So my vote would be to disable it, at least per default

Reply:
@misl6 it looks to me like `SUPPORT_MP3_SMPEG := false` is still removed in your pull request. Is that intentional? Did your flags revert go wrong?

Reply:
@AndreMiras for what it's worth, this is currently set to `false`/disabled on master. So removing this diff probably **enables** it in this pull request - which I think would be problematic because of the licensing. I think it's better left out, LGPL in `.apk` isn't an unsolvable problem but can bite developers who are unaware this will be included

Reply:
@misl6 @AndreMiras oops, poked around some more and SMPEG was just removed entirely from SDL2_Mixer, so false alarm :smile: sorry for the comment spam regarding this particular flag